catalog/resource_metastore_test.go

```diff
@@ -36,7 +36,10 @@ func TestCreateMetastore(t *testing.T) {
 				Method:   "PATCH",
 				Resource: "/api/2.0/unity-catalog/metastores/abc",
 				ExpectedRequest: map[string]interface{}{
-					"owner": "administrators",
+					"owner":                 "administrators",
```

Comment:
one last thing before we merge this - can you leave `TestCreateMetastore` as it was and create new test with delta sharing fields?